MeshSync_URP~/Packages/com.unity.meshsync/package.json

```diff
@@ -14,7 +14,8 @@
         "com.unity.modules.particlesystem": "1.0.0",      
         "com.unity.modules.screencapture": "1.0.0",      
         "com.unity.sharp-zip-lib": "1.3.0-preview",
-        "com.unity.timeline": "1.2.18"     
+        "com.unity.timeline": "1.2.18",
```

Comment:
So, am I correct to think that this PR is just for adding some new packages, like mathematics, to `MeshSync_URP~` project because perhaps you need to write some local debugging code in this project ?

Reply:
Yeah mostly to add probuilder and the splines package, splines has a dependency on mathematics but I cannot add splines because it's an internal package and the official release doesn't support what we need.